content/src/controller/Controller.ts

```diff
@@ -717,13 +716,7 @@ export enum DeploymentField {
   AUDIT_INFO = 'auditInfo'
 }
 
-export type ControllerPointerChanges = Omit<DeploymentPointerChanges, 'changes'> & {
-  changes: {
-    pointer: Pointer
-    before: EntityId | undefined
-    after: EntityId | undefined
-  }[]
-}
+export type ControllerPointerChanges = Omit<DeploymentPointerChanges, 'changes'>
```

Comment:
I think the `ControllerPointerChanges` is not necessary right now? Or at least the `omit` thing?

Reply:
Or the thing to be omitted is the new field `pointers`?

Reply:
I've removed the ControllerPointerChanges as now it's the same as DeploymentPointerChanges